doc/varnish-configuration.rst

```diff
@@ -341,6 +349,13 @@ request with :ref:`a proper user hash <return context hash>`.
     set the ``req.url`` to a fixed URL. Otherwise Varnish would cache every
     hash lookup separately.
 
+    The ``fos_user_context_hash`` should be used to separate the cache of the
+    hash lookup. If you don't do that, Varnish can run into performance issues
```

Comment:
maybe link [the code change](https://github.com/varnishcache/varnish-cache/pull/3520/files#diff-09291b164556649a3f302cac0242fe37da79c35c2e9bb06bd1866536689f4ac0R137] or the original PR?

Reply:
good idea, i link that PR now